src/Parsers/getInsertQuery.cpp

```diff
@@ -19,10 +19,8 @@ std::string getInsertQuery(const std::string & db_name, const std::string & tabl
         query.columns->children.emplace_back(std::make_shared<ASTIdentifier>(column.name));
 
     WriteBufferFromOwnString buf;
-    IAST::FormatSettings settings(buf, true);
-    settings.always_quote_identifiers = true;
-    settings.identifier_quoting_style = quoting;
-    query.IAST::format(settings);
+    IAST::FormatSettings settings(true, true, quoting);
```

Comment:
Fixed. A few notes:

Con: modern IDE should be able to inlay parameter names.
Pro: emacs [seems to have acquired](https://lists.gnu.org/archive/html/emacs-devel/2023-02/msg00841.html) such functionality just a month ago, and GitHub doesn't have it at all.

Con: now we can't have any c-tor for the struct (aggregate class shouldn't have any user-defined c-tors).
Pro: there's only one usage which needed a separate c-tor and it could be done differently.

Con: it's too verbose if initializing with variables: `.one_line = one_line`
Pro: inlay parameter names suggested by IDE aren't smart enough to avoid verboseness, too :(

All in all, the designated initializers win here :)

![image](https://user-images.githubusercontent.com/5155395/229646252-a91e5aed-3d1b-493c-992a-4580dd29bca5.png)